lab-05/linked-list.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+class Node{
+  constructor(value) {
+    this.value = value;
+    this.next = null
+  }
+}
+
+class LinkedList {
+  constructor(node) {
+    this.head = node.value;
```

Comment:
The head of the linked list is a reference to the entire Node (not just the node's value)